core/pom.xml

```diff
@@ -50,6 +50,14 @@
       <version>1.1.1</version>
     </dependency>
     <dependency>
+      <groupId>com.google.code.findbugs</groupId>
+      <artifactId>jsr305</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.google.errorprone</groupId>
+      <artifactId>error_prone_annotations</artifactId>
+    </dependency>
+    <dependency>
```

Comment:
Sorry the jsr305 was a spurious change.  Without the error-prone annotations dependency, I see errors of the form:

```
error: cannot access CanIgnoreReturnValue
  class file for com.google.errorprone.annotations.CanIgnoreReturnValue not found
```